src/courtroom.cpp

```diff
@@ -3188,7 +3189,7 @@ void Courtroom::handle_song(QStringList *p_contents)
     }
 
     if (!mute_map.value(n_char)) {
-      chatlogpiece *temp = new chatlogpiece(str_char, str_show, f_song, true,
+      chatlogpiece *temp = new chatlogpiece(str_char, str_show, f_song, tr("has played a song"),
```

Comment:
it's back

Reply:
fuck

Reply:
wait @oldmud0 





code duplication